client/src/views/classes/Class.js

```diff
@@ -89,7 +87,20 @@ export default function Class (props) {
     </>
   )
 };
+const displayTime=(duration)=>{
 
+  if(duration<1){
+    return `${Math.round(duration*60)} mins`;
+  }
+  else{
+    if(duration%1===0){
+      return `${duration} hour`;
+    }
+    else{
+      return `${duration} hour ${Math.round(duration*60)} mins`;
+    }
+  }
+};
```

Comment:
Fantastic work on this.  Great job!

Reply:
I just realized I made a mistake. It should be 
line 101: return `${Math.floor(duration)} hour ${Math.round(duration%1*60)} mins`;
I will fix it.